src/App.tsx

```diff
@@ -15,35 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Empty = '',
+  Length = 'id',
+  Alphabetically = 'alphabetically',
```

Comment:
why `id`? It's better to match value and key if there is no reason to write something else (as with `Empty`)
```suggestion
  Length = 'Length',
  Alphabetically = 'Alphabetically',
```